include/MemoryModel/MutablePointsToDS.h

```diff
@@ -75,19 +59,19 @@ class MutablePTData : public PTData<Key, KeySet, Data, DataSet>
 
     virtual inline bool addPts(const Key &dstKey, const Data& element) override
     {
-        addSingleRevPts(revPtsMap[element], dstKey);
+        if (this->rev) addSingleRevPts(revPtsMap[element], dstKey);
```

Comment:
can we put this `if` branch inside `addSingleRevPts ` or `addRevPts `

Reply:
Done.